### ISSUE TITLE
Webhooks not received for certain domains

### ISSUE THREAD

**Issue by passuf** [wunderlist/api#35]
I played around with webhooks for some time and created a small application which I deployed on heroku, everything worked liked expected. When I moved the application to my own server, I did not receive the webhooks anymore. I am still able to communicate with the API and are able to create and receive new webhooks e.g. with a URL pointing to a Runscope Bucket, but if I add my own domain as URL, the webhook is created, but I do not receive any call when updating the corresponding list.

If I forward the request with Runscope, my application receives and handles the webhook, so my endpoint should be working correctly. According to my server logs there are no https or http calls coming from wunderlist, even though there are webhooks registered with the correct URL pointing to my application.

Now I am wondering if a domain first needs to be white listed or similar before receiving webhooks, since heroku.com and runscope.com are well known services but [my domain](https://wunderhabit.passuf.ch) is of course not.

Thanks a lot for your help!

**Comment by gdiab** [comment]
were you getting events on completed tasks from webhooks? I only got events for a task being created but not for being completed. this was all local dev using runscope.

**Comment by passuf** [comment]
@gdiab Yes, I receive an event when a task is completed. In fact, I receive an event each time the task is updated. The `operation` of the event is `update`, and there are a `before` and `after` object to see the changes made to the task. When completed, the key `is_completed` changes from `false` to `true`.

**Comment by ashishjen** [comment]
I am facing similar issue and not getting Webhook updates even with an http endpoint. As passuf mentioned I can see that the update is coming on requestb.in but not to my domain.

**Comment by qadirsuh** [comment]
Mee too facing this issue, I am creating a webhook for my localhost, like 
`http://localhost:8080/list_webhook`
but its not getting invoked on create new task in a certain list event.

**Comment by passuf** [comment]
@qadirsuh I do not know whether you can subscribe a webhook to events such as reaching a reminder date/time. Alternatively you could save the reminder time locally when the task is updated (your webhook gets invoked), and then you store and process these notifications on your server, e.g. using a simple cron-job.
